src/realm/group.cpp

```diff
@@ -832,7 +832,7 @@ void Group::remove_table(size_t table_ndx, TableKey key)
     // tables. Such a behaviour is deemed too obscure, and we shall therefore
     // require that a removed table does not contain foreign origin backlink
     // columns.
-    if (table->is_cross_table_link_target())
+    if (!ignore_backlinks && table->is_cross_table_link_target())
```

Comment:
indeed. we remove all tables so it should not cause any issue. I don't think we expose this to the SDKs (had a look in `spec.yml` and there is no reference). I could also have a private method and friend the class (or some other utility)

Reply:
Java was the only SDK which exposed table schema mutations directly and everything else only does it by setting the object store schema. Should probably add a comment in the declaration that `ignore_backlinks=true` will leave things in an invalid state just in case someone ends up using it in the future.

Reply:
good point